src/lib/transaction_snark/transaction_snark.ml

```diff
@@ -277,6 +277,92 @@ module Base = struct
     in
     Boolean.Assert.any [Boolean.not is_user_command; verifies]
 
+  let check_timing ~account ~txn_amount ~txn_global_slot =
```

Comment:
Could you add a test checking that the unchecked and checked versions of check_timing match up?

Reply:
something besides what's in the test module `account timing check` in this file?